src/main/resources/graphql/flashcards.graphqls

```diff
@@ -58,9 +58,9 @@ type FlashcardProgressDataLog {
 
 type FlashcardSide {
     """
-    Text of this flashcard side. This can be markdown.
+    Text of this flashcard side as ResourceMarkdown.
     """
-    text: String!
+    text: ResourceMarkdown!
```

Comment:
If we name this text, probably everyone would expect a string with just text. Wouldn't it make sense to rename this field? "content" or whatever

Reply:
I just dislike the double "text" nesting. But whatever, I don't have an idea. You could just call it "resourceMarkdown" or "markdown" I guess 😅